lang/cs/Org.Apache.REEF.IMRU/OnREEF/Driver/IAllContextsReceived .cs

```diff
@@ -0,0 +1,27 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+using Org.Apache.REEF.Utilities.Attributes;
+
+namespace Org.Apache.REEF.IMRU.OnREEF.Driver
+{
+    [Private]
+    internal interface IAllContextsReceived
```

Comment:
It'll work either way, but OnNext(int) reads like a counter.... have to read through comments and code to understand what the intent is.
I think typed event makes sense here.

---
In reply to: [67778447](https://github.com/apache/reef/pull/1033#discussion_r67778447) [](ancestors = 67778447)